features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Apis.jsx

```diff
@@ -16,22 +16,42 @@
  * under the License.
  */
 
-import React from 'react'
-import {Route, Switch} from 'react-router-dom'
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
 
-import Listing from './Listing/Listing'
-import Details from './Details/index'
-import {PageNotFound} from '../Base/Errors'
+import APIListing from './Listing/APIListing';
+import APIProductListing from './Listing/APIProductListing';
+import Details from './Details/index';
+import { PageNotFound } from '../Base/Errors';
 
-
-class Apis extends React.Component {
+/**
+ * Default API Store overview page
+ *
+ * @class Apis
+ * @extends {PureComponent}
+ */
+class Apis extends React.PureComponent {
```

Comment:
Shall we make this a functional component ?

Reply:
+1